### DISCUSSION TITLE
Konfigurasi prettier

### DISCUSSION THREAD

**Discussion by farhanreizha** [deaafrizal/cuyuniverse-forum#117] [Q&A] [answered]
A kalo misalnya di tambahin configure prettier biar formatt code nya sama mulai dari indentasi, kutip 2 / 1 sama titik koma gmna ? Biar ngga terjadi conflict juga dari code nya soalnya kan ada beberapa kofigurasi prettier orang beda2 tuh.

**Comment by deaafrizal** [comment, marked as answer]
> A kalo misalnya di tambahin configure prettier biar formatt code nya sama mulai dari indentasi, kutip 2 / 1 sama titik koma gmna ? Biar ngga terjadi conflict juga dari code nya soalnya kan ada beberapa kofigurasi prettier orang beda2 tuh.

setuju euy biar gak conflict cuma gara2 baris code doang. coba di setup han, jangan lupa pull dari dev trhir

**Reply by farhanreizha** [reply]
oke siap
